chart/kubeapps/Chart.yaml

```diff
@@ -25,4 +25,4 @@ maintainers:
 name: kubeapps
 sources:
   - https://github.com/kubeapps/kubeapps
-version: 7.0.2-dev1
+version: 7.0.3-dev1
```

Comment:
If the upstream chart is now 7.0.3, and we've just synced those changes and added our dev changes on top (great), shouldn't the version be a > 7.0.3 but < 7.0.4 ? That is, I expected version to be 7.0.4-dev1 . But if that complicates things, no problem.

Reply:
Ooops, you're totally right, my bad :P
Changing it to 7.0.4-dev1 right awway.